securedrop/tests/functional/functional_test.py

```diff
@@ -33,19 +34,50 @@ class FunctionalTest():
 
     def _unused_port(self):
         s = socket.socket()
-        s.bind(("localhost", 0))
+        s.bind(('127.0.0.1', 0))
         port = s.getsockname()[1]
         s.close()
         return port
 
+
+    def _create_tor_browser_webdriver(self, abs_log_file_path):
+        # Read the TorBrowser location from Ansible
```

Comment:
`expanduser` fails for some reason related to the fact that TBB changes directories when it creates a driver.  I found a workaround using `getpass.getuser()` though.